test/test_interpolation.py

```diff
@@ -107,9 +105,6 @@ def test_project_same_space_mixed_adjoint(self):
         P1 = FunctionSpace(self.source_mesh, "CG", 1)
         Vs = P1 * P1
         source = Function(Vs)
-        s1, s2 = source.subfunctions
-        s1.interpolate(self.sinusoid())
-        s2.interpolate(-self.sinusoid())
```

Comment:
@jwallwork subfunctions don't seem to raise a warning so I did not need to make a change here, but lint raised an error `F841 local variable 's1' is assigned to but never used` so I had to delete these lines. Does this work as intended?